src/app/components/client/system-report-modal/system-report-form.vue

```diff
@@ -1,5 +1,24 @@
 <template>
 	<app-form name="systemReportForm">
+		<app-form-group name="issue_tracker_url" optional :label="$gettext('Issue Tracker Link')">
+			<p class="help-block">
+				<translate>
+					Paste the link to your ticket here. It's optional, but without it we can't get back to you
+					if we need more info to solve it.
+				</translate>
+				<span class="emoji emoji-huh" />
+			</p>
+
+			<app-form-control
+				type="text"
+				placeholder="https://github.com/gamejolt/issue-tracker/issues/1313"
+				:rules="{
+					max: 255,
```

Comment:
Probably should also have a min length (of 42, which would be the length of this: `github.com/gamejolt/issue-tracker/issues/`), or a regex to test for that url shape.